packages/vm/examples/module_size.rs

```diff
@@ -35,14 +34,15 @@ pub fn main() {
     println!("wasm size: {wasm_size} bytes");
 
     // Compile module
-    let module = module_compile(&wasm);
+    let engine = make_compiling_engine(None, &[]);
+    let module = compile(&engine, &wasm).unwrap();
```

Comment:
This `compile` helper is now a little redundant / useless. I would say, let's either remove it, or use it everywhere instead of the explicit `Module` creation.

Reply:
I kept it to hide some internals in one of the examples. Not sure anymore. I use it now in all places.